aws/modules/deploy/helms/cert-manager-values.yml

```diff
@@ -0,0 +1,3 @@
+chart-metadata:
```

Comment:
This reads better to me, thanks.

I still don't like the name of this file. How about cert-manager-metadata.yml or cert-manager-gpii-infra.yml or something to indicate that this is a special, non-Helm facility.